src/_data/vbe.yml

```
@@ -0,0 +1,190 @@
+versions:
+  - 2.3.0
+  - 2.3.1
+  - 2.3.2
+  - 2.3.3
+  - 2.3.4
+extension:
```

```suggestion
extensions:
```
conventionally, plural will help to identify that you should expect an array when you want to parse data

---

agreed. i'll update the data file and template and push the change.